Ethics.Rmd

```diff
@@ -0,0 +1,35 @@
+# Ethical Considerations
```

Comment:
Can we link to this page somewhere in the main text?

Reply:
Maybe we can include it under "How will this new policy affect me?". We could say something like "in addition to considering whether the new NIH policy requires you to create a data sharing plan, you should also consider some of the ethical considerations. A more detailed discussion can be found in the Appendix" or something.

Reply:
yeah I have some thoughts about this... I will add to the PR

Reply:
actually I will add these to a new PR so we have the link